estate/models/estate_property.py

```diff
@@ -0,0 +1,107 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api, exceptions
+from odoo.tools.float_utils import float_is_zero, float_compare
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = 'Estate Property'
+    _order = "id desc"
+
+    name = fields.Char('Title', required=True, default='Unknown', translate='True')
+    active = fields.Boolean('Active', default=True)
+    description = fields.Text('Description')
+    postcode = fields.Char('Postcode')
+    date_availability = fields.Date('Available From', copy=False, default=fields.Date.add(fields.Date.today(), months=3))
+    expected_price = fields.Float('Expected Price', required=True)
+    selling_price = fields.Float('Selling Price', readonly=True, copy=False)
+    bedrooms = fields.Integer('Bedrooms', default=2)
+    living_area = fields.Integer('Living Area (sqm)')
+    facades = fields.Integer('Facades')
+    garage = fields.Boolean('Garage')
+    garden = fields.Boolean('Garden')
+    garden_area = fields.Integer('Garden Area (sqm)')
+    garden_orientation = fields.Selection(string='Garden orientation',
+                                          selection=[('north', 'North'),
+                                                    ('south', 'South'),
+                                                    ('east', 'East'),
+                                                    ('west', 'West')]
+                                          )
+    state = fields.Selection(string='State',
+                            selection=[('new', 'New'),
+                                    ('offer_received', 'Offer Received'),
+                                    ('offer_accepted', 'Offer Accepted'),
+                                    ('sold', 'Sold'),
+                                    ('cancelled', 'Cancelled')],
+                             default='new', required=True, copy=False)
```

Comment:
Same, the indent is a little bit weird, it looks better to just have one argument per line here
```suggestion
    state = fields.Selection(
        string='State',
        selection=[
            ('new', 'New'),
            ('offer_received', 'Offer Received'),
            ('offer_accepted', 'Offer Accepted'),
            ('sold', 'Sold'),
            ('cancelled', 'Cancelled'),
        ],
        default='new',
        required=True,
        copy=False,
    )
```